test/unit/spec/LoginRouter_spec.js

```diff
@@ -385,6 +387,27 @@ function (Okta, Q, Backbone, SharedUtil, CryptoUtil, CookieUtil, OktaAuth, Util,
         expect(form.hasErrors()).toBe(false);
       });
     });
+    itp('navigates to PrimaryAuth if status is UNAUTHENTICATED', function () {
+      return setup({ stateToken: 'aStateToken' })
+      .then(function (test) {
+        Util.mockRouterNavigate(test.router);
+        test.setNextResponse(resUnauthenticated);
+        test.router.navigate('');
```

Comment:
Am curious if this will immediately-ish resolve (without checking for anything) since we're starting on PrimaryAuth. Maybe for testing purposes, it might be better to start out on a different page.